product_barcode_constraint_per_company/tests/test_module.py

```diff
@@ -1,37 +1,82 @@
-# Copyright (C) 2019 - Today: GRAP (http://www.grap.coop)
```

Comment:
Please check carefully, the init file is in the repository. It is not displayed because this file is immutable
![image](https://github.com/OCA/stock-logistics-barcode/assets/43703214/9d11de58-9189-4441-85b7-1c1be66941b1)

Reply:
sorry, I mixed w/ the other init file in this PR :v: